contracts/test/ReserveV2.sol

```diff
@@ -7,21 +7,18 @@ import "../rsv/ReserveEternalStorage.sol";
  * @dev A version of the Reserve Token for testing upgrades.
  */
 contract ReserveV2 is Reserve {
-    constructor() public {
```

Comment:
We are able to remove the constructor because `Reserve` now begins paused as well.